setup.py

```diff
@@ -53,6 +56,17 @@ def build_extension(self, ext: Extension) -> None:
         build_envd_if_not_found()
 
 
+class bdist_wheel_abi3(bdist_wheel):
+    def get_tag(self):
+        python, abi, plat = super().get_tag()
+
+        if python.startswith("cp"):
+            # on CPython, our wheels are abi3 and compatible back to 3.6
+            return "cp36", "abi3", plat
```

Comment:
Also seems "none" instead of "abi3" is more concise here since we don't depend on python abi. Ref: https://github.com/pypa/wheel/blob/main/src/wheel/bdist_wheel.py#L295